src/index.js

```diff
@@ -1490,6 +1493,17 @@ export default class ReactJkMusicPlayer extends PureComponent {
           }
           this.audioListsPlay(audioLists[currentPlayIndex + 1][PLAYER_KEY])
         } else {
+          if (restartCurrentOnPrev) {
+            if (this.audio.currentTime <= 1) {
```

Comment:
Please add restartCurrentOnPrev unit test case in `__tests__/tests/player.test.js`

Reply:
Actually, I don't have much experience writing tests. Can you help me out with this?